### PR TITLE
Update wavefront-sdk-python version to 1.8.0

### PR DESCRIPTION
**Pull request by joannatk**
`wavefront-sdk-python` v1.7.9 Sends internal diagnostic metrics as delta counters.

### CODE REVIEW

**Review by haosong — COMMENTED** [review]
Let's wait until `v1.7.9` is available: https://pypi.org/project/wavefront-sdk-python/.
And pls bump the version in `setup.py` and license file as well.

File: setup.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 setuptools.setup(
     name='wavefront-pyformance',
-    version='1.2.0',
+    version='1.3.0',
```
Comment (ustinov) [inline setup.py]: Looks like a minor update here, shouldn't it be `1.2.1`?

**Comment by ustinov** [comment]
@joannak-vmware updating `wavefront-sdk-python` to `1.8`+ is bringing lots of deprecation warnings when using `wavefront-pyformance`, do we want to fix those?

**Comment by joannatk** [comment]
@ustinov , yes I think we should fix them since we are updating to `wavefrront-sdk-python 1.8.0` in this PR.

File: wavefront_pyformance/wavefront_reporter.py [hunk 2]
```diff
@@ -185,9 +188,14 @@ def __init__(self, server, token, source='wavefront-pyformance',
         self.server = self._validate_url(server)
         self.token = token
         self.batch_size = 10000
-        self.wavefront_client = wavefront_sdk.WavefrontDirectClient(
-            self.server, token, batch_size=self.batch_size,
+
+        client_factory = WavefrontClientFactory()
+        client_factory.add_client(
+            url="https://{}@{}".format(token, urlparse(server).hostname),
```
Comment (haosong) [inline wavefront_pyformance/wavefront_reporter.py]: Actually we probably shouldn't parse to put only hostname. There's case such as use Direct ingestion to send to "localhost:8080", dropping the port will fail such usecase

**Review by haosong — APPROVED** [review]
One comment, the rest LGTM

**Review by ustinov — APPROVED** [review]

**Pull request merged**